### PR TITLE
Improve savings transaction overview visuals

### PR DESCRIPTION
**Pull request by oskarvu**
Closes FRO-711

### CODE REVIEW

File: packages/app/src/features/dialogs/savings/common/types.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sDaiToken: Token
+  sDaiBalanceBefore: NormalizedUnitNumber
+  sDaiBalanceAfter: NormalizedUnitNumber
+  APY: Percentage
+  outTokenAmount: NormalizedUnitNumber
+}
+
+export type SavingsDialogTxOverview =
+  | {
+      showExchangeRate: boolean
```
Comment (yivlad) [inline packages/app/src/features/dialogs/savings/common/types.ts:17]: Extract to TxOverview?

File: packages/app/src/features/dialogs/savings/deposit/logic/createTxOverview.ts [hunk 2]
```diff
@@ -34,29 +24,33 @@ export function createTxOverview({
...
       sDaiBalanceBefore,
       sDaiBalanceAfter,
       APY: savingsInfo.apy,
+      outTokenAmount: sDaiAmount,
     }
   }
 
-  if (!swapInfo.data) {
-    return undefined
+  const showExchangeRate = swapParams.fromToken.symbol !== 'DAI'
```
Comment (yivlad) [inline packages/app/src/features/dialogs/savings/deposit/logic/createTxOverview.ts:50]:
```suggestion
  const showExchangeRate = swapParams.fromToken.symbol !== marketInfo.DAI.symbol
```

Comment (yivlad) [reply packages/app/src/features/dialogs/savings/deposit/logic/createTxOverview.ts:50]: Same in other places

Comment (oskarvu) [reply packages/app/src/features/dialogs/savings/deposit/logic/createTxOverview.ts]: Need to double check copilot suggestions ;)

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**